### ISSUE TITLE
Provide Some Example

### ISSUE THREAD

**Issue by SakibFarhad** [zafartahirov/fingerpi#3]
Hello, 
It will be a great help if you could provide some documentation or some example as I am have some problem with Enroll, It is not clear to me how use them. 

Thank You

**Comment by zafartahirov** [comment]
The `FingerPi` package is just a UART interface and a bridge to the scanner from Python. The actual documentation is provided by the manufacturer. I don't have the `Enroll` method per se, you will have to make it yourself. The reason is that `Enroll` is a 4 step process, and you will have to check the `ACK` signal after every single one of them. In addition to that, the step 2, 3, 4 require for you to check if the finger is present and after each enrollment released. Also, you need initialization for the ID... So the whole process is not atomic (see below)

If you read the [documentation](http://cdn.sparkfun.com/datasheets/Sensors/Biometric/GT-511C3_datasheet_V2.1_20161025.pdf), page 18 describes what responses you should await from every request, while page 34 describes the enrollment protocol

Basically it boils down to you sending these atomic requests:

1. Check if `ID` is taken (`ID` has to be new) using `CheckEnrolled`
2. `EnrollStart(ID)`
3. Wait to place finger using`IsFingerPressed`
4. `CaptureFinger`
5. `Enroll1`
6. Wait to take off finger using `IsFingerPressed`
7. Wait to place finger using`IsFingerPressed`
8. `CaptureFinger`
9. `Enroll2`
10. Wait to take off finger using `IsFingerPressed`
11. Wait to place finger using`IsFingerPressed`
12. `CaptureFinger`
13. `Enroll3`

**Comment by SakibFarhad** [comment]
Thanks a lot sir, I think I got it.
